pkg/artifactory/resource_artifactory_access_token_test.go

```diff
@@ -19,7 +19,7 @@ func TestAccAccessTokenAudienceBad(t *testing.T) {
 			email = "existinguser@a.com"
 			admin = false
 			groups = ["readers"]
-			password = "Passsword1"
+			password = "Passsw0rd!"
```

Comment:
Important change

Reply:
Artifactory 7.37.13 adds the new requirement for 1 special character. Some of our tests fail so need updating.